src/gluonts/model/deepar/_estimator.py

```diff
@@ -489,3 +451,23 @@ def create_predictor(
             ctx=self.trainer.ctx,
             dtype=self.dtype,
         )
+
+    def get_schema(self) -> Schema:
+        fields: Dict[str, Any] = dict()
+        default_values: Dict[str, Any] = dict()
```

Comment:
Minor: I prefer `{}` over `dict`.

Also, do you need to declare the types for `mypy` to not complain?